BetaCameras/BaslerToF/BaslerToF.cs

```diff
@@ -332,14 +334,14 @@ protected override void UpdateImpl()
 
                     if (IsChannelActive(ChannelNames.Distance))
                     {
-                        depthImage = new FloatCameraImage(width, height);
+                        distanceImage = new FloatCameraImage(width, height);
                         for (int y = 0; y < height; y++)
                         {
                             for (int x = 0; x < width; x++)
                             {
                                 Coord3D c = bufferPoint3f[y * width + x];
```

Comment:
We should use i++ as with the other channels

Reply:
done

Reply:
Nice